### PR TITLE
Fix tiles view preview images in general (zooming issue) and files list thumbnails in spaces  

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes [Spaces overview: Thumbnail image is zoomed in · Issue #9328 · owncloud/web](https://github.com/owncloud/web/issues/9328)
- Fixes [[Bug] distorted/clinched space thumbnails in list view · Issue #10133 · owncloud/web](https://github.com/owncloud/web/issues/10133)

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/src/components/Spaces/SpaceHeader.vue [hunk 1]
```diff
@@ -209,18 +200,9 @@ export default defineComponent({
...
         }
-        const decodedUri = decodeURI(props.space.spaceImageData.webDavUrl)
-        const webDavPathComponents = decodedUri.split('/')
-        const idComponent = webDavPathComponents.find((c) => c.startsWith(props.space.id as string))
-        if (!idComponent) {
-          return
-        }
-        const path = webDavPathComponents
-          .slice(webDavPathComponents.indexOf(idComponent) + 1)
-          .join('/')
```
Comment (JanAckermann) [inline packages/web-app-files/src/components/Spaces/SpaceHeader.vue:220]: i don't feel this is necessary, its hard coded everywhere else anyways

File: packages/web-app-files/src/views/spaces/GenericSpace.vue [hunk 2]
```diff
@@ -664,11 +670,16 @@ export default defineComponent({
 
       const debounced = debounce(({ unobserve }) => {
         unobserve()
+        const processor =
+          this.viewMode === ViewModeConstants.tilesView.name
+            ? ProcessorType.enum.fit
+            : ProcessorType.enum.thumbnail
```
Comment (dschmidt) [inline packages/web-app-files/src/views/spaces/GenericSpace.vue:675]: would it make sense to add a `.processorType` prop to the view mode constants?

File: packages/web-app-files/src/views/spaces/Projects.vue [hunk 3]
```diff
@@ -383,6 +395,35 @@ export default defineComponent({
...
+      }
+
+      const resource = await (clientService.webdav as WebDAV).getFileInfo(space, {
+        path: `.space/${space.spaceImageData.name}`
+      })
+
+      const processor =
+        unref(viewMode) === ViewModeConstants.tilesView.name
+          ? ProcessorType.enum.fit
+          : ProcessorType.enum.thumbnail
```
Comment (dschmidt) [inline packages/web-app-files/src/views/spaces/Projects.vue:415]: would also reduce the code here

File: packages/web-app-files/src/views/spaces/GenericSpace.vue (hunk 2 above)
Comment (dschmidt) [reply packages/web-app-files/src/views/spaces/GenericSpace.vue:675]: we quickly discussed it and agreed, we don't want to do it right now. 

I will probably touch this code soon to make it possible to define viewModes as extensions, will revisit it then.

**Review by kulmann — COMMENTED** [review]
Needs a rebase, but looks good to me so far 👍

File: packages/web-app-files/src/views/spaces/Projects.vue [hunk 4]
```diff
@@ -495,7 +521,7 @@ export default defineComponent({
   height: 75vh;
 }
 
-.table-preview {
+F .table-preview {
```
Comment (kulmann) [inline packages/web-app-files/src/views/spaces/Projects.vue:524]:
```suggestion
.table-preview {
```

**Review by kulmann — APPROVED** [review]

**Pull request merged**